db/indexes.go

```diff
@@ -387,10 +382,7 @@ func removeObsoleteIndexes(bucket base.Bucket, previewOnly bool, useXattrs bool)
 
 	// Attempt removal of candidates, adding to set of removedIndexes when found
 	for _, indexName := range removalCandidates {
-		removed, removeError := removeObsoleteIndex(gocbBucket, indexName, previewOnly)
-		if removeError != nil {
-			return removedIndexes, removeError
-		}
+		removed, _ := removeObsoleteIndex(bucket, indexName, previewOnly)
```

Comment:
Review whether we should be logging error here.  We could optionally fail silently on the case where the error was 'index does not exist', but should log any other errors.